tracer/build/_build/Build.GitHub.cs

```diff
@@ -348,7 +348,6 @@ partial class Build
 
             var sb = new StringBuilder();
             sb.AppendLine($"⚠ 1. Download the NuGet packages for the release from [this link]({artifactsLink}) and upload to nuget.org");
-            sb.AppendLine("⚠ 2. Download the signed MSI assets and native symbols from GitLab and attach to this release before publishing");
```

Comment:
I don't think this is necessary either now right? You fixed that in this PR! 🎉

Reply:
Done in 1ad8ac42a336fecd7aca60a84fe6b98aca0109d7